foundation/src/main/scala/org/scalameta/ast/ast.scala

```diff
@@ -250,7 +250,7 @@ class AstMacros(val c: Context) {
       // step 10: finish codegen for Quasi
       qstats1 += q"def pt: _root_.java.lang.Class[_] = _root_.org.scalameta.runtime.arrayClass(_root_.scala.Predef.classOf[$iname], this.rank)"
       if (is("Term.Block.Quasi") || is("Type.Bounds.Quasi") || is("Ctor.Primary.Quasi") ||
-          is("Import.Clause.Quasi") || is("Lit.String.Quasi") || is("Mod.Annot.Quasi")) {
+          is("Import.Clause.Quasi") || is("Lit.String.Quasi")) {
```

Comment:
Yes. Otherwise `[error] scala.NotImplementedError: implementation restriction: dangerous quasi, see the sources of @ast for more information`